src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -9,29 +9,21 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{2B96AA10-84C0-4927-8611-8D2474B990E8}</ProjectGuid>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <OmitResources Condition="'$(TargetGroup)' != 'netfx'">true</OmitResources>
+    <OmitResources>true</OmitResources>
```

Comment:
Delete this line and also `src\System.Globalization.Extensions\src\Resources\Strings.resx`